packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx

```diff
@@ -70,7 +70,13 @@ const AddWalletButton = ({ device, instances, addDeviceInstance, selectDeviceIns
                               })
                     }
                 >
-                    <Translation id="TR_ADD_WALLET" />
+                    {/* check if users has standard wallet enabled 
+                     (device.useEmptyPassphrase === true) means that standard wallet is enabled */}
+                    {device.useEmptyPassphrase ? (
```

Comment:
this doesn't work the way you want it to 😁 
You are changing the label based on currently selected wallet (device === wallet in this case)
<img width="869" alt="Screenshot 2020-10-29 at 10 32 57" src="https://user-images.githubusercontent.com/6961901/97550656-3f414200-19d2-11eb-93ea-e8dbad0507d5.png">

Reply:
Aaah got it :grin: Thank you, I will fix that

Reply:
It should be OK now (see the last screenshot)